renovate.json5

```diff
@@ -1,15 +1,15 @@
 {
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   extends: [
+    "config:recommended",
```

Comment:
The `config:recommended` moves around between commits.

Reply:
Ahh, got it. I wanted to move it up, but I guess the formatter is sorting the lists.